File: src/javascript/administration/route/search-routes.mjs
import { FaxCoreClient, printResult } from '../../common/faxcoreClient.mjs';

const client = new FaxCoreClient();

const result = await client.post('/api/route/search', {
  input: process.env.FAXCORE_ROUTE_SEARCH || ''
});

printResult('Inbound route search', result);

File: src/dotnet/FaxCore.ApiSamples/Samples/Administration/Route/SearchRoutes.cs
using System.Threading.Tasks;

namespace FaxCore.ApiSamples.Samples.Administration.Route
{
    internal static class SearchRoutes
    {
        public static Task<string> RunAsync(FaxCoreClient client)
        {
            return client.PostAsync("/api/route/search", new { input = string.Empty });
        }
    }
}

